dev/TeachingTip/TeachingTip.cpp

```diff
@@ -529,72 +530,73 @@ bool TeachingTip::PositionUntargetedPopup()
     // Depending on the effective placement mode of the tip we use a combination of the tip's size, the window's size, and the target
     // offset property to determine the appropriate vertical and horizontal offsets of the popup that the tip is contained in.
     auto&& popup = m_popup.get();
+    auto isLeftToRight = FlowDirection() == winrt::FlowDirection::LeftToRight;
     switch (PreferredPlacement())
```

Comment:
Same comment here. If all usages of finalTipWidth are affected by isLeftToRight, you can just reverse its sign once.

Reply:
This one is less enticing, for Near and Far placements we want 0 in one case and Positive in the other, while in center placement we want negative.

---
In reply to: [270512149](https://github.com/Microsoft/microsoft-ui-xaml/pull/504#discussion_r270512149) [](ancestors = 270512149)